CMakeLists.txt

```
@@ -1,103 +1,96 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.13.4)
 project(tf2_2d)
 
-add_compile_options(-std=c++14)
-add_compile_options(-Wall)
-add_compile_options(-Werror)
-
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  tf2
-  tf2_geometry_msgs
-)
+find_package(ament_cmake REQUIRED)
+find_package(Boost REQUIRED)
 find_package(Eigen3 REQUIRED)
-
-catkin_package(
-  INCLUDE_DIRS
-    include
-  CATKIN_DEPENDS
-    roscpp
-    tf2
-    tf2_geometry_msgs
-  DEPENDS
-    EIGEN3
-)
+find_package(rclcpp REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
 
 ###########
 ## Build ##
 ###########
 
-# Header-only package. Nothing to build.
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Werror)
+endif()
+
+add_library(tf2_2d INTERFACE)
+target_compile_features(tf2_2d INTERFACE cxx_std_14)
+target_include_directories(tf2_2d INTERFACE
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>")
+target_link_libraries(tf2_2d INTERFACE
+  Boost::boost
+  Eigen3::Eigen
+  rclcpp::rclcpp
+  tf2::tf2
+  tf2_ros::tf2_ros
+  ${tf2_geometry_msgs_TARGETS})
```

As of https://github.com/ros2/geometry2/commit/9e94db43b2e3d232b5083cc4374317c0b76fee30 there's a target

```suggestion
  tf2_geometry_msgs::tf2_geometry_msgs)
```

---

Ah! I assumed it was an interface package. Fixed in c1f4ffe